GenerateNU/website
# Fix resizing for FP and Interested

# What was the ticket?
Fix mobile views for FP and Interested panels

# What did I do?
- Change logic for which view to show for resizing to show correct view for mobile
   - Created new hook to check for whether a view is mobile or not on window resize

# How did I test it?
- Check various mobile sizes to make sure views are still good
- Check desktop sizes to make sure the resizing fixes stayed
 
# Additional comments for the reviewers
Sometimes there are issues if you switch from mobile to desktop with the wrong view being shown. However, I don't think this matters too much in production because someone's device isn't going to switch from mobile to desktop. If I do the debugger and put breakpoints, it works, but it doesn't always if I just switch it.
 
# Screenshots
MY VERSION
<img width="366" alt="image" src="https://user-images.githubusercontent.com/15839864/232237553-513acd2b-dd21-43ca-8ce9-275dd196d4d9.png">
<img width="366" alt="image" src="https://user-images.githubusercontent.com/15839864/232237561-2cf7f80e-32ca-4dff-a7ef-970035fce77b.png">
<img width="366" alt="image" src="https://user-images.githubusercontent.com/15839864/232237569-bb11b2e6-0fe0-405b-987b-14afda297527.png">
<img width="366" alt="image" src="https://user-images.githubusercontent.com/15839864/232237573-ba659176-978f-4f5a-9b91-4e926b4f8e76.png">
